### PR TITLE
update arm support for Amazon EKS integration

### PR DESCRIPTION
**Pull request by linhlam-kc**
Update specific instruction to install Kubecost on EKS cluster with ARM-based processor (Amazon Graviton2).

Related issue: https://github.com/kubecost/cost-analyzer-helm-chart/issues/1672

### CODE REVIEW

**Comment by dwbrown2** [comment]
@bstuder99 are you ok to start reviewing newly submitted docs PRs?

**Comment by bstuder99** [comment]
@dwbrown2 Yeah I've been tagged in a lot of misc. doc fixes before, this PR looks good

File: aws-eks-cost-monitoring.md [hunk 1]
```diff
@@ -33,6 +33,18 @@ oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
 -f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml
 ```
 
+To install Kubecost on Amazon EKS cluster on AWS Graviton2 (ARM-based processor), you can run following command:
+
+```bash
+helm upgrade -i kubecost \
+oci://public.ecr.aws/kubecost/cost-analyzer --version 1.96.0 \
```
Comment (dwbrown2) [inline aws-eks-cost-monitoring.md:40]: Should we be consistent with command above and use $VERSION? I'm good either way, but seems like being consistent here.

**Review by dwbrown2 — APPROVED** [review]
LGTM, with one comment

File: aws-eks-cost-monitoring.md (hunk 1 above)
Comment (linhlam-kc) [reply aws-eks-cost-monitoring.md:40]: It was my typo when copy and paste from my testing note. I will update to $VERSION before merging tomorrow. Thank you for your review.

**Pull request merged**